util/src/main/java/io/kubernetes/client/pager/Pager.java

```diff
@@ -137,4 +85,53 @@ public String getContinueToken() {
       return continueToken;
     }
   }
+
+  private class PagerIterator implements Iterator<ApiType> {
+
+    private boolean started;
+    private String continueToken;
+    private Call call;
+    private int offsetCurrentPage;
+    private int currentPageSize;
+
+    /**
+     * returns false if kubernetes server has exhausted List.
+     *
+     * @return
```

Comment:
Either fill in more details or delete this. (and below too)

Let's include JavaDoc tags if we're using them, but delete them if not.

Reply:
updated and filled docs